### ISSUE TITLE
Tool to report new device

### ISSUE THREAD

**Issue by rhwood** [rhwood/paducah#1]
This project needs a bash script that wraps `udevadm monitor`, `udevadm info`, and curl to observe TTY devices as they are added to the system running the script, checks with the web service for matches, lists matches, and offers to upload the device if no match is found, or if no match is correct.

**Comment by rhwood** [comment]
This tool is named *paducah.sh* and has the following parameters:
```
--device=DEVICE Get information about DEVICE
--all-rules Get rules for all known devices
--info Display device information
--quiet Take default actions without prompting
  This is the same as: --all-rules --upload
--rules Generate rules for device
--no-rules Do not generate rules for device
--upload Upload device information
--no-upload Do not upload device information
--version Display the version and copyright
--help Display this message
```
If parameters are not specified this tool will guide the user through determining the device and creating a rule. No other actions than those described above are anticipated for it.

**Comment by rhwood** [comment]
Blocked by #6

**Comment by rhwood** [comment]
Due to #14, the upload new device story for this tool is changing:

The new story is that this tool will upload the udevadm output and provide a link to complete the upload in a browser. If the user has a GitHub account, the user will be able to complete the upload, otherwise, the user will need to create a GitHub account to complete the device upload. With a GitHub account, the user can fill out incomplete information and submit the device for review.

**Comment by rhwood** [comment]
Complete except for:
- [ ] need to verify it wraps values in quotes when submitting an issue correctly
- [ ] ensuring messages from submitting an issue are presented nicely to user (don't dump JSON)
